src/crop_image.py

```diff
@@ -1,69 +1,122 @@
-#!/usr/bin/env python
 
+'''
+Created on March 15th, 2018
+author: Michael Rodriguez
+sources: https://www.blog.pythonlibrary.org/2017/10/03/how-to-crop-a-photo-with-python/
+description: Crops images of a given object class to feed into neural network.
+'''
+# !/usr/bin/env python
 from os import listdir
```

Comment:
I don't see the purpose importing a bunch of os functions, if you are still just going to import os